### PR TITLE
Move rust code that exists for our tests to its own module

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

**Comment by alex** [comment]
I hate this module name. So suggestions welcome.

**Comment by reaperhulk** [comment]
`nose` 🧌

**Comment by alex** [comment]
lmao. could do something like `test_support`?

File: src/rust/src/test_support.rs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+// 2.0, and the BSD License. See the LICENSE file in the root of this repository
+// for complete details.
+
+use crate::error::CryptographyResult;
+use asn1::SimpleAsn1Readable;
+use cryptography_x509::certificate::Certificate;
+use cryptography_x509::common::Time;
+use cryptography_x509::name::Name;
+
+#[pyo3::pyclass(frozen, module = "cryptography.hazmat.bindings._rust.pytests")]
```
Comment (reaperhulk) [inline src/rust/src/test_support.rs:11]:
```suggestion
#[pyo3::pyclass(frozen, module = "cryptography.hazmat.bindings._rust.test_support")]
```

**Review by reaperhulk — APPROVED** [review]

**Pull request merged**